resources/js/Pages/Results.vue

```diff
@@ -330,8 +354,22 @@ h2 {
     }
 }
 
-.form-buttons {
-    text-align: end;
-    margin-top: $dimension-layout-xsmall;
+.form-footer {
+    padding-top: $dimension-layout-xsmall;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    min-height: 4.75em;
```

Comment:
Where is this value coming from? One should be really careful when using `em` since it is relative to the font-size of the element it is placed. If a font size will be set on this footer at some point, then the 1 em base will be of that font size, which can cause some confusion as to how the height is calculated.

Reply:
Well, the purpose of this setting is to not make the table below "jump" when the message shows up. I thought that a box size relative to the font size actually makes sense, because with a different font size we'd need a different box height. I also confirmed by manually changing the font-size in css.

As for the value  itself, I think I found it with try-and-error in the browser, for the case that the message actually has two lines. What would be a better approach or unit or value instead?

Reply:
Replaced by a calculation for a two-line message box.